docs/getting-started/developer.md

```diff
@@ -40,221 +39,167 @@
         # you should see a "Build succeeded"
         ```
 
-    5. Save the following file as `atlasinfo.txt`
+    5. Save the following as `atlasinfo.txt`
```

Comment:
```suggestion
    5. Save the following as `atlasinfo.txt` or copy the `atlasinfo.txt` from your client root.epk
```

Reply:
I didn't knew `atlasinfo.txt` exists on the client :O I wrote it myself :D